.babelrc

```diff
@@ -20,7 +20,8 @@
         ],
         "node": "6.0"
       }
-    }]
+    }],
+    "stage-0"
```

Comment:
Why did you need to bring this in? Not necessarily against it, but I think it has larger implications for the sdk build steps

Reply:
This allows babel to support the "spread" operator.  It allows a syntax like this:
```
obj = {
  foo: source.fooey,
  ...(source.barey && {bar: source.barey})
};
```
Which results in an object that has no "bar" key if there is no "barey" in the source.
This syntax isn't NECESSARY, but the alternative is a bunch of if statements after the object definition, ie:

```
obj = {
 foo: source.fooey
};
if (source.barey) {
  obj.bar = source.barey;
}
```

Not terrible but if you have many keys that you want to evaluate the code gets much longer

Reply:
The widgets are using the ["transform-object-rest-spread"](https://www.npmjs.com/package/babel-plugin-transform-object-rest-spread) to accomplish this. Might make the change less intrusive to use that instead.

Reply:
Ack, confirmed this works.

Reply:
@adamweeks this is in the latest commit now.